setup.py

```diff
@@ -554,23 +547,18 @@ def run(self):
         ],
         data_files=[
             ("%s" % webconfig, ["build/config/apache/cobbler.conf"]),
-            ("%s" % webconfig, ["build/config/apache/cobbler_web.conf"]),
             ("%s/templates" % libpath, glob("autoinstall_templates/*")),
             ("%s/templates/install_profiles" % libpath, glob("autoinstall_templates/install_profiles/*")),
             ("%s/snippets" % libpath, glob("autoinstall_snippets/*", recursive=True)),
             ("%s/scripts" % libpath, glob("autoinstall_scripts/*")),
             ("%s" % libpath, ["config/cobbler/distro_signatures.json"]),
-            ("share/cobbler/web", glob("web/*.*")),
-            ("%s" % webcontent, glob("web/static/*")),
-            ("%s" % webimages, glob("web/static/images/*")),
             ("share/cobbler/bin", glob("scripts/*.sh")),
             ("share/cobbler/web/templates", glob("web/templates/*")),
             ("%s/webui_sessions" % libpath, []),
```

Comment:
```suggestion
            ("share/cobbler/web/templates", glob("web/templates/*")),
            ("%s/webui_sessions" % libpath, []),
```
What about those two? Then line 351 has to be adjusted, too.

Reply:
I am not sure. Let's leave it like this and see after some refactorings if that is needed or not.